src/dapps/ens-manager-dapp/TheENSManagerLayout.vue

```diff
@@ -544,7 +573,10 @@ export default {
     this.detactUrlChangeTab();
 
     const ens = this.network.type.ens
-      ? new ENS(this.web3.currentProvider, this.network.type.ens.registry)
+      ? new ENS({
+          provider: this.web3.eth.currentProvider,
+          ensAddress: getEnsAddress(this.network.type.chainID.toString())
```

Comment:
you can get the ens address from the network object, cz I think getEnsAddress might not have all chainIds

Reply:
`this.network.type.ens.registry`